config/cloud.cfg.tmpl

```diff
@@ -305,6 +320,11 @@ system_info:
 {% elif variant == "openmandriva" %}
   network:
     renderers: ['network-manager', 'networkd']
+{% elif variant == "raspberry-pi-os" %}
+  network:
+    dhcp_client_priority: [dhclient]
```

Comment:
Upstream dhclient is abandoned. Is raspberry-pi-os's plan to support it forever?

Reply:
This was included as an error, and dhclient is not actually used by Raspberry Pi OS - instead, we use network-manager's built in DHCP client. I believe this has been corrected in the latest version of this PR.